doc/mediasdk-man.md

```diff
@@ -5671,11 +5672,14 @@ The `mfxPlatform` structure contains information about hardware platform.
 --- | ---
 `CodeName` | Intel® microarchitecture code name. See the [PlatformCodeName](#PlatformCodeName) enumerator for a list of possible values.
 `DeviceId` | Reserved.
+`MediaAdapterType` | Description of Intel Gen Graphics adapter type. See the [mfxMediaAdapterType](#mfxMediaAdapterType) enumerator for a list of possible values.
 
 **Change History**
 
 This structure is available since SDK API 1.19.
 
+The SDK API **TBD** adds `MediaAdapterType` field.
```

Comment:
@onabiull : how we will handle these in releases if API won't land in the release? I would suggest to make API changes in separate commits which we can specially mark somehow and just revert commits in release branch.

Reply:
My plan is to strip all \*\*TBD\*\* sections from documents in release branch. Probably in the same commit with version change. After all there are just dozen or so of them.

Reply:
So, manual editing of docs?) Shouldn't we hide hints for automatic cut inside md comments for TBD sections?

Reply:
Right now - yes. We can think about employing html-style comments (they won't be rendered), but I not sure that in current scheme - 4 releases per year, it would worth investments.